tests/test_downloader.py

```diff
@@ -402,6 +402,81 @@ def test_oc_series_found(self):
                 log_message = cm.output[-1].split(":")[-1]
                 assert log_message == expected_msg
 
+    def test_upload_to_s3(self):
```

Comment:
Make sure we can actually successfully ignore a deleted file link and still have a successful ingest.